Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
     .library(name: "Commander", targets: ["Commander"]),
   ],
   dependencies: [
-    .package(url: "https://github.com/kylef/Spectre.git", from: "0.9.2"),
+    .package(url: "https://github.com/kylef/Spectre.git", .revision("69da3ff"))
```

Comment:
I think, it's worth to bump Spectre's version number.

Reply:
Yeah. I've merged this for now. I plan to release both (and I'll jump this back to a version number when I do so).